Fabric.Identity.API/scripts/Register.ps1

```diff
@@ -24,7 +24,8 @@ function Get-ApplicationUrl($serviceName, $serviceVersion, $discoveryServiceUrl)
     $discoveryResponse = Invoke-RestMethod -Method Get -Uri $discoveryRequest -UseDefaultCredentials
     $serviceUrl = $discoveryResponse.value.ServiceUrl
     if([string]::IsNullOrWhiteSpace($serviceUrl)){
-        throw "The service $serviceName version $serviceVersion is not registered with the Discovery service. Make sure that this version of the service is registered w/ Discovery service before proceeding. Halting installation."
+	    Write-Error "There was an error registering the service $serviceName, using DiscoveryService url $discoveryRequest."
```

Comment:
I think this error should say something about "There was an error getting the service registrations for $serviceName using DiscoveryService url $discoveryRequest. I don't think we are registering with discovery here, simply trying to retrieve the discovery registration.